yannickl88/features-bundle/1.0/etc/packages/features.yaml

```diff
@@ -0,0 +1,21 @@
+features:
+    tags:
+#        my.feature:
+#            request: ["beta", "on"]
+
+services:
```

Comment:
In this case, the goal is to make sure services that depend on this package are removed when removing the package, else you would have to remove them manually.

It seems fine to me :/